charts/team-ns/templates/_ingress.tpl

```diff
@@ -13,7 +13,7 @@
 {{- $v := .dot.Values }}
 {{- $istioSvc := print "istio-ingressgateway-" .type }}
 {{- $cm := index $v.apps "cert-manager" }}
-{{- $tlsSecretName := ternary $cm.externallyManagedTlsSecretName "otomi-cert-manager-wildcard-cert" (or (eq $cm.issuer "byo-wildcard-cert") (eq $cm.issuer "externally-managed-tls-secret")) }}
+{{- $derived := index $v "_derived" }}
```

Comment:
since derive is always available you do not need index function.

The reference `$v._derived`  is perfectly fine.